pkg/api/resource/quantity_proto.go

```diff
@@ -69,7 +69,7 @@ func (m *Quantity) Size() (n int) {
 	_ = l
 
 	// BEGIN CUSTOM SIZE
-	l = len(m.String())
+	l = m.Len()
```

Comment:
Size is only ever called with marshaling.  This doesn't seem to reduce the number of allocations because now you allocate size twice?